### ISSUE TITLE
Don't flag slnt as hidden

### ISSUE THREAD

**Issue by davelab6** [googlefonts/roboto-flex#304]
The `slnt` axis in v3.002 uses the "HIDDEN_AXIS" flag in the "fvar" table (as described in https://docs.microsoft.com/en-us/typography/opentype/spec/fvar) but it should not. 

The other axes' flags are OK :)

Was

```
    <!-- slnt -->
    <Axis>
      <AxisTag>slnt</AxisTag>
      <Flags>0x1</Flags>
      <MinValue>-10.0</MinValue>
      <DefaultValue>0.0</DefaultValue>
      <MaxValue>0.0</MaxValue>
      <AxisNameID>260</AxisNameID>
    </Axis>
```

Should be

```
    <!-- slnt -->
    <Axis>
      <AxisTag>slnt</AxisTag>
      <Flags>0x0</Flags>
      <MinValue>-10.0</MinValue>
      <DefaultValue>0.0</DefaultValue>
      <MaxValue>0.0</MaxValue>
      <AxisNameID>260</AxisNameID>
    </Axis>
```

**Comment by m4rc1e** [comment]
We have the following for all axes:

```XML
  <axes>
    <axis tag="wght" name="wght" minimum="100" maximum="1000" default="400">
      <labelname xml:lang="en">wght</labelname>
    </axis>
    <axis tag="wdth" name="wdth" minimum="25" maximum="151" default="100">
      <labelname xml:lang="en">wdth</labelname>
    </axis>
    <axis tag="opsz" name="opsz" minimum="8" maximum="144" default="14">
      <labelname xml:lang="en">opsz</labelname>
      <map input="8" output="-1"/>
      <map input="14" output="0"/>
      <map input="36" output="0.492000"/>
      <map input="84" output="0.946000"/>
      <map input="144" output="1"/>
    </axis>
    <axis tag="GRAD" name="GRAD" minimum="-200" maximum="150" default="0">
      <labelname xml:lang="en">GRAD</labelname>
    </axis>
    <axis tag="slnt" name="slnt" minimum="-10" maximum="0" default="0" hidden="1">
      <labelname xml:lang="en">slnt</labelname>
    </axis>
    <axis tag="XTRA" name="XTRA" minimum="323" maximum="603" default="468" hidden="1">
      <labelname xml:lang="en">XTRA</labelname>
    </axis>
    <axis tag="XOPQ" name="XOPQ" minimum="27" maximum="175" default="96" hidden="1">
      <labelname xml:lang="en">XOPQ</labelname>
    </axis>
    <axis tag="YOPQ" name="YOPQ" minimum="25" maximum="135" default="79" hidden="1">
      <labelname xml:lang="en">YOPQ</labelname>
    </axis>
    <axis tag="YTLC" name="YTLC" minimum="416" maximum="570" default="514" hidden="1">
      <labelname xml:lang="en">YTLC</labelname>
    </axis>
    <axis tag="YTUC" name="YTUC" minimum="528" maximum="760" default="712" hidden="1">
      <labelname xml:lang="en">YTUC</labelname>
    </axis>
    <axis tag="YTAS" name="YTAS" minimum="649" maximum="854" default="750" hidden="1">
      <labelname xml:lang="en">YTAS</labelname>
    </axis>
    <axis tag="YTDE" name="YTDE" minimum="-305" maximum="-98" default="-203" hidden="1">
      <labelname xml:lang="en">YTDE</labelname>
    </axis>
    <axis tag="YTFI" name="YTFI" minimum="560" maximum="788" default="738" hidden="1">
      <labelname xml:lang="en">YTFI</labelname>
    </axis>
  </axes>
```

Are there any others which shouldn't be hidden?

**Comment by m4rc1e** [comment]
Ah, just seen that the other axes are ok. Ignore the above.

**Comment by m4rc1e** [comment]
Fixed in ea4abfcf1263c1496bd3ffd84a8f00e83aa8b28f
